ark/utils/data_utils_test.py

```diff
@@ -49,12 +49,21 @@ def test_save_fov_images():
         for fov in fovs_sub:
             assert os.path.exists(os.path.join(temp_dir, fov + '.tiff'))
 
-    # test 3: name suffix provided
+    # test 3: name suffix provided, along with sub_dir provided
     with tempfile.TemporaryDirectory() as temp_dir:
-        data_utils.save_fov_images(fovs_sub, temp_dir, sample_img_xr, name_suffix='_test')
+        data_utils.save_fov_images(fovs_sub, temp_dir, sample_img_xr, sub_dir="_test",
```

Comment:
I would pick a different name for the sub_dir, right now it looks like there's a link between the `name_suffix` and `sub_dir` when in reality they're unrelated

Reply:
These are both still named the same thing?